src/core/annotation.js

```diff
@@ -514,6 +514,13 @@ class Annotation {
     this._isOffscreenCanvasSupported = params.isOffscreenCanvasSupported;
     this._fallbackFontDict = null;
     this._needAppearances = false;
+
+    // The annotation cannot be changed (neither its position/visibility nor its
+    // contents), hence we can just display its appearance and don't generate
+    // a HTML element for it.
+    const isLocked = !!(this.flags & AnnotationFlag.LOCKED);
+    const isContentLocked = !!(this.flags & AnnotationFlag.LOCKEDCONTENTS);
+    this.data.noHTML = isLocked && isContentLocked;
```

Comment:
Can we please just move this code up a little bit, such that we can directly set `noHTML: isLocked && isContentLocked,` in the `this.data` object?